package.json

```diff
@@ -7,12 +7,13 @@
     "renovate-config-validator": "bin/config-validator.js"
   },
   "scripts": {
-    "build": "run-s clean compile:* copy-static-files",
-    "build:docker": "run-s compile:ts copy-static-files",
+    "build": "run-s clean generate:* compile:* copy-static-files",
+    "build:docker": "run-s generate:* compile:ts copy-static-files",
     "clean": "rimraf dist",
     "clean-cache": "node bin/clean-cache.js",
     "compile:ts": "tsc -p tsconfig.app.json",
     "compile:dts": "tsc -p tsconfig.dts.json",
+    "generate:imports": "node --experimental-modules tools/generate-imports.js",
     "copy-static-files": "copyfiles -u 1 -e **/__fixtures__/**  -e **/__mocks__/** lib/**/*.json lib/**/*.py dist/",
     "create-json-schema": "babel-node --extensions \".ts,.js\" -- bin/create-json-schema.js && prettier --write \"renovate-schema.json\"",
     "debug": "babel-node --inspect-brk  --extensions \".ts,.js\" -- lib/renovate.ts",
```

Comment:
`"test": "run-s lint test-schema generate:imports type-check jest"`

Reply:
I'm on fixing build, thinking doing the build before lint.